From: SeongJae Park <sj@kernel.org>
Date: Sun, 22 Dec 2024 14:49:25 -0800
Subject: [PATCH] ==== fixup online commit behavior and memory leaks ====

mm/damon/core: fix memory leaks and ignored inputs from damon_commit_ctx()

Due to two bugs in damon_commit_targets() and damon_commit_schemes(),
which are called from damon_commit_ctx(), some user inputs can be
ignored, and some mmeory objects can be leaked.  Fix those.

Note that only DAMON sysfs interface users are affected.  Other DAMON
core API user modules that more focused more on simple and dedicated
production usages, including DAMON_RECLAIM and DAMON_LRU_SORT are not
using the buggy function in the way, so not affected.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/iVb1QZlI | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/iVb1QZlI

diff --git a/damon_meta_changes/iVb1QZlI b/damon_meta_changes/iVb1QZlI
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

